### PR TITLE
fixed issues for selection input position

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR for #1243 #1230 

Fixed selection input position,
And added clear button for clear 

Before:
Screenshot 2023-10-10 at 2 59 47 PM
After:
Screenshot 2023-10-10 at 2 58 29 PM

### CODE REVIEW

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 17 min

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 13 min

File: django_project/frontend/src/containers/MainPage/SideBar/Filter.tsx [hunk 1]
```diff
@@ -570,150 +569,156 @@ function Filter(props: any) {
         }
     };
 
+    const ClearFilter = () => {
```
Comment (dimasciput) [inline django_project/frontend/src/containers/MainPage/SideBar/Filter.tsx:572]:
```suggestion
    const clearFilter = () => {
```

File: django_project/frontend/src/containers/MainPage/SideBar/Filter.tsx [hunk 2]
```diff
@@ -570,150 +569,156 @@ function Filter(props: any) {
...
-                        isOptionEqualToValue={(option, value) => option.id === value.id}
-                    />
-                </Box>
                 )}
-                {
-                    allowOrganisationSelection && <Box>
+                <Box className='sidebarBoxHeading'>
+                    <img src="/static/images/species/Elephant.svg" alt='species image' />
+                    <Typography color='#75B37A' fontSize='medium'>Species</Typography>
+                    <div className='clear-button'><Typography color='#75B37A' className='pointer' fontSize='medium' onClick={ClearFilter}>Clear</Typography></div>
```
Comment (dimasciput) [inline django_project/frontend/src/containers/MainPage/SideBar/Filter.tsx:677]:
```suggestion
                    <div className='clear-button'><Typography color='#75B37A' className='pointer' fontSize='medium' onClick={clearFilter}>Clear</Typography></div>
```

**Review by dimasciput — CHANGES_REQUESTED** [review]
Please also fix the conflicts

**Comment by dimasciput** [comment]
The filter was updated with another PR

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** FAILURE
**Build duration:** 41 sec

**Pull request closed without merging**